pymeasure/instruments/keithley/keithley2281S.py

```diff
@@ -0,0 +1,325 @@
+
+import enum
+
+from pymeasure.instruments import Instrument
+from pymeasure.instruments.keithley.buffer import KeithleyBuffer
+from pymeasure.instruments.validators import strict_discrete_set, strict_range, truncated_range
+import numpy as np
+import pandas as pd
+
+
+@enum.unique
+class Keithley2281SOperationCondition(enum.IntFlag):
+    """Enum containing Keithley2281S operation condition defintition"""
+    CALIBRATION = 1
+    _RESERVED_1 = 2
+    _RESERVED_2 = 4
+    _RESERVED_3 = 8
+    MEASUREMENT = 16
+    TRIGGER = 32
+    ARM = 64
+    _RESERVED_4 = 128
+    FILT = 256
+    LIST = 512
+    IDLE = 1024
+    _RESERVED_5 = 2048
+    _RESERVED_6 = 4096
+    _RESERVED_7 = 8192
+    _RESERVED_8 = 16384
+
+
+class Keithley2281S(Instrument, KeithleyBuffer):
+    """
+    Represents the Keithley 2281S-20-6 power supply and battery simulator / characterizer.
+    Common commands, beside `function_mode` and power supply commands should also work for
+    Keithley 2280S power supplies, although this is untested.
+    """
+    _VOLTAGE_RANGE = [0.0, 20]
+    _CURRENT_RANGE_PS = [0.1, 6.1]
+    _CURRENT_RANGE_BT_BS = [0.0, 6.1]
+    _INTERNAL_MEMORY_SLOTS = [*range(1, 10)]
+
+    def __init__(self, adapter, name="Keithley2281S", **kwargs):
+        super().__init__(adapter, name, **kwargs)
+
+    # Common commands
+
+    cm_display_text_data = Instrument.setting(
+        ":DISP:USER:TEXT '%s'",
+        """Control text to be displayed(24 characters)."""
```

Comment:
A property's docstring should start with "Set" for a `setting`, "Control" for a `control` and "Get" or "Measure" for a `measurement`.